terraform/main.tf

```diff
@@ -267,6 +267,16 @@ variable "stackdriver_exporter_helm_chart_version" {
   default = ""
 }
 
+variable "single_object_validation_batch_localities" {
```

Comment:
nit: Expressing this in this manner opens the possibility of typos, where we might have `ingestors.ingestor-1.localities.gondor`, but set `single_object_validation_batch_localities["gnodor"]`. We could instead add a boolean valued field to the `ingestors.localities` object. However what you've got here is consistent with how we did the key rotation localities so either is fine with me.